### ISSUE TITLE
Lights, camera, action

### ISSUE THREAD

**Issue by sczerwinski** [glimpse-graphics/glimpse-framework#45]
It would be nice to render scene with some code like this:

``` kotlin
lights {
    // …
}
camera {
    // …
}
action {
    sphere madeOf redPlastic
    cube madeOf texturedMaterial
}
```

**Comment by sczerwinski** [comment]
The _lights, camera, action_ pattern should be encapsulated inside a _[scene](https://en.wikipedia.org/wiki/Scene%5F%28filmmaking%29)_.

Scenes will be further combined into _[sequences](https://en.wikipedia.org/wiki/Sequence%5F%28filmmaking%29)_, after the animations are implemented in issue #13.

**Comment by sczerwinski** [comment]
Descoped
